src/foam/nanos/bench/benchmarks/FUIDBenchmark.java

```diff
@@ -0,0 +1,39 @@
+/**
+ * @license
+ * Copyright 2021 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package foam.nanos.bench.benchmarks;
+
+import foam.core.X;
+import foam.dao.*;
+import foam.nanos.auth.User;
+import foam.nanos.bench.Benchmark;
+import foam.util.UIDGenerator;
+import foam.nanos.app.AppConfig;
+
+public class FUIDBenchmark
+  implements Benchmark
+{
+  protected UIDGenerator generator_ = new UIDGenerator();
+
+  @Override
+  public void setup(X x) {
+  }
+
+  @Override
+  public void teardown(X x, java.util.Map stats) {
+  }
+
+  @Override
+  public void execute(X x) {
+    AppConfig config = (AppConfig) x.get("appConfig");
+
+    if ( config.getMode() == foam.nanos.app.Mode.PRODUCTION ) {
+      return;
+    }
+
+    generator_.generate();
```

Comment:
Benchmark benchmarking overhead will be more expensive than the cost of a single operation. To fix, do much more work for each invocation. ie. generate 1000 IDs instead of 1.

Reply:
Actually, I just fixed the BenchmarkRunner so you don't need to do this:

commit 520edc823dc560b28ad5b6a1c84a7f365316ca31
Author: Kevin Greer <kgrgreer@gmail.com>
Date:   Wed Jul 7 15:46:04 2021 -0400

    Reduce benchmark accounting overhead for short-running benchmarks with use an invocationCount.